README.md

```diff
@@ -107,60 +124,96 @@ iis_site { 'complete':
     },
   ],
   require => File['c:\\inetpub\\complete'],
-}
+} ->
 
 iis_virtual_directory { 'vdir':
   ensure       => 'present',
   sitename     => 'complete',
   physicalpath => 'c:\\inetpub\\complete_vdir',
-  require => File['c:\\inetpub\\complete_vdir'],
+  require      => File['c:\\inetpub\\complete_vdir'],
 }
 ```
 
 ## Reference
 
 ### Facts
 
-* `iis_version` - The version of the installed IIS. Empty string if not installed.
+* `iis_version` - The version of IIS that is installed. Empty string if not installed.
+
 
-### Types/Providers
+### Types and Providers
 
+* [iis_feature](#iis_feature)
 * [iis_application](#iis_application)
 * [iis_application_pool](#iis_application_pool)
-* [iis_feature](#iis_feature)
 * [iis_site](#iis_site)
+* [iis_virtual_directory](#iis_virtual_directory)
+
+
+### iis_feature
+
+Allows installation and removal of IIS Features.
+
+#### Properties/Parameters
+
+##### `ensure`
+
+Must be either 'present' or 'absent'.
+
+##### `name`
+
+The name of the feature.
+
+##### `include_all_subfeatures`
+
+Indicates whether to install all subfeatures of the feature. For instance, ASP.NET as well as IIS Web Server.
+
+##### `restart`
+
+Indicates whether to allow a restart if the feature installation requests one.
+
+##### `include_management_tools`
+
+Indicates whether to automatically install all management tools for the feature.
+
+##### `source`
+
+Optionally include a source path for the installation media for the feature.
+
 
 ### iis_application
 
-Allows creation of a new IIS application.
+Allows creation of a new IIS Application and configuration of application parameters.
+
+#### Properties/Parameters
 
 #### `ensure`
 
-Must be either 'present' or 'absent'. Present will ensure the application pool is created.
+Must be either 'present' or 'absent'. Present will ensure the application is created.
 
 #### `applicationname`
 
-The name of the Application. The virtual path of an application is '/<applicationname>'.
+The name of the application. The virtual path of the application is '/<applicationname>'.
```

Comment:
Microsoft and their darned caps, man. "application" feels like it could go either way, depending on whether we are using as the generic noun or the MS-specific proper name Application. In this case, I'm inclined to defer to your Microsoft expertise on this one.